test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -473,7 +446,7 @@ protected bool CheckOutputIsSuffix(string basePath, string outPath, int skip = 0
         }
 
         protected bool CheckEqualityFromPathsCore(string relPath, string basePath, string outPath, int skip = 0,
-            int digitsOfPrecision = DigitsOfPrecision, NumberParseOption parseOption = NumberParseOption.Default)
+            int digitsOfPrecision = DigitsOfPrecision, NumberParseOption parseOption = NumberParseOption.Default, bool allowMismatch = false)
```

Comment:
This function is named CheckEqualityFromPathsCore. This function shouldn't be called by functions that need to check for unequal values. So this function shouldn't take an allowMismatch parameter.